services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowCrudV2Spec.groovy

```diff
@@ -518,35 +518,301 @@ class FlowCrudV2Spec extends HealthCheckSpecification {
         Wrappers.wait(WAIT_OFFSET) { assert !islUtils.getIslInfo(newIsl).isPresent() }
     }
 
-    @Ignore("https://github.com/telstra/open-kilda/issues/2576")
-    @Tags(VIRTUAL)
-    def "System doesn't allow to create a one-switch flow on a DEACTIVATED switch"() {
-        given: "A deactivated switch"
+    @Unroll
+    def "Able to CRUD #flowDescription one-switch pinned flow"() {
+        when: "Create a flow"
         def sw = topology.getActiveSwitches().first()
+        def flow = flowHelperV2.singleSwitchFlow(sw)
+        flow.maximumBandwidth = bandwidth
+        flow.ignoreBandwidth = bandwidth == 0
+        flow.pinned = true
+        flowHelperV2.addFlow(flow)
+
+        then: "Pinned flow is created"
+        def flowInfo = northbound.getFlow(flow.flowId)
+        flowInfo.pinned
+
+        when: "Update the flow (pinned=false)"
+        northbound.updateFlow(flowInfo.id, flowInfo.tap { it.pinned = false })
+
+        then: "The pinned option is disabled"
+        def newFlowInfo = northbound.getFlow(flow.flowId)
+        !newFlowInfo.pinned
+        flowInfo.lastUpdated < newFlowInfo.lastUpdated
+
+        and: "Cleanup: Delete the flow"
+        flowHelper.deleteFlow(flow.flowId)
+
+        where:
+        flowDescription | bandwidth
+        "a metered"     | 1000
+        "an unmetered"  | 0
```

Comment:
Please remove iterations. It doesn't matter whether we install meter or not when talking about 'pinned' flag